ambry-store/src/main/java/com.github.ambry.store/StoreFindToken.java

```diff
@@ -0,0 +1,238 @@
+/**
```

Comment:
Actual changes in this file are the capitalization of the static constants and the implementation of `toBytes()`.